source/postprocess/visualization/depth.cc

```diff
@@ -73,7 +73,7 @@ namespace aspect
                                                   "depth",
                                                   "A visualization output postprocessor that outputs "
                                                   "the depth for all points inside the domain, as "
-                                                  "determined by the geometry model."
+                                                  "determined by the initial surface of the geometry model. "
```

Comment:
I think this is a bit more precise, as the initial surface could have already been deformed (mesh deformation can create initial topography).
```suggestion
                                                  "determined by the undeformed surface of the geometry model. "
```